Sources/UberCore/Authentication/Tokens/KeychainUtility.swift

```diff
@@ -145,6 +149,12 @@ public final class KeychainUtility: KeychainUtilityProtocol {
         ) == noErr
     }
     
+    /// Updates the accessGroup used for subsequent operations
+    /// - Parameter accessGroup: The new accessGroup
+    public func setAccessGroup(_ accessGroup: String) {
```

Comment:
can we update the API to receive more than just the key instead? maybe a struct with the key and access group

Reply:
I was mostly keeping parity with the legacy API since it made migration easier, but I can update to accept this as a parameter with a default nil value.

Reply:
Updated. Please take a look @renan-ssoares